test/verify/check-login

```diff
@@ -167,7 +155,10 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
                       "echo 'echo noise-profile-out; echo noise-profile-err >&2' >> .profile")
             self.addCleanup(m.execute, "set -e; cd ~admin; mv .bashrc.bak .bashrc; "
                                        "if [ -e .profile.bak ]; then mv .profile.bak .profile; else rm .profile; fi")
-            login("admin", "foobar")
+            b.try_login("admin", "foobar")
+            m.execute("echo 'echo noise-rc-out; echo noise-rc-err >&2' >> ~admin/.bashrc")
+            m.execute("echo 'echo noise-profile-out; echo noise-profile-err >&2' >> ~admin/.profile")
+            b.try_login("admin", "foobar")
```

Comment:
What does that do? I can't see any difference to what it alreaady does  above, except that it's now writing the noise twice?

Reply:
This might have been a rebase fail.  I'll look into it.